CHANGELOG.md

```diff
@@ -1,98 +1,53 @@
-## 4.0.0 (YYYY-MM-DD)
+## 4.0.0-RC1 (YYYY-MM-DD)
 
 ## Breaking Changes
 
-* Calling `distinct()` on a sorted `RealmResults` no longer clears the sorting (#3503).
+The internal fileformat has been upgraded. Opening an older Realm will upgrade the file automatically, but older versions of Realm will can no longer be able to read the file.  
```

Comment:
`fileformat` -> `file format`. also please mention the file format version, should be version 9 i think. 
https://github.com/realm/realm-core/commit/a43e8449d3a20e37ae63d9564e825bcbfdad372f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR13

Reply:
I cannot find any mention of the fileformat version in Cores changelog. With that in mind, it seems a bit odd that we put it in ours?

Reply:
The link i posted is not correct?

> Changing instruction values for `Table::move_row()` requires a version bump to 9.
>  Version 8 files in read only mode without any history can be opened without 

 
 But yeah, i guess it is OK to not mention it ...